src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/action/delete/DeleteTransformsRequest.kt

```diff
@@ -0,0 +1,32 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.delete
+
+import org.elasticsearch.action.ActionRequest
+import org.elasticsearch.action.ActionRequestValidationException
+import org.elasticsearch.action.ValidateActions.addValidationError
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+class DeleteTransformsRequest(
+    val ids: List<String>
+) : ActionRequest() {
+
+    @Throws(IOException::class)
+    constructor(sin: StreamInput) : this(
+        ids = sin.readStringList()
+    )
+
+    override fun validate(): ActionRequestValidationException? {
+        var validationException: ActionRequestValidationException? = null
+        if (ids.isEmpty()) {
+            validationException = addValidationError("List of ids to delete is empty", validationException)
```

Comment:
can you add a test case for this, by setting the REST request id to be empty string